src/container/styles.scss

```diff
@@ -11,6 +11,7 @@
   @include styles.styles-reset;
   word-wrap: break-word;
   position: relative;
+  overflow: hidden;
```

Comment:
We cannot do this, because it will make absolute elements (dropdowns and popover) cropped by the container borders.

Reply:
That's so true. I'll update.
Another option that I was thinking but with a hesitation is to reduce the border-radius size from the media wrapper `.media` like `border-top-left-radius: calc(awsui.$border-radius-container - 1px);`.